integration/localnet/Makefile

```diff
@@ -24,6 +24,8 @@ COMMIT=$(shell git rev-parse HEAD)
 VERSION=localnetbuild
 
 CURRENT_DIRECTORY=$(shell pwd)
+TPS_INIT=25
```

Comment:
can you add a comment here explaining the purpose of these vars and how they'r only used for load testing?

Reply:
Done.